salt/pillar/nodegroups.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+'''
+=================
+Nodegroups Pillar
+=================
+
+Introspection: to which nodegroups does my minion belong?
+Provides a pillar with the default name of `nodegroups`
+which contains a list of nodegroups which match for a given minion.
+
+Command Line
+------------
+
+.. code-block:: bash
+    salt-call pillar.get nodegroups
+    local:
+        - class_infra
+        - colo_sj
+        - state_active
+        - country_US
+        - type_saltmaster
+
+Configuring Nodegroups Pillar
+-----------------------------
+
+.. code-block:: yaml
+
+    ext_pillar:
```

Comment:
Is there supposed to be a value in `ext_pillar`? Or should the `extenstion_modules` part be bumped in a bit? Just making sure this example is correct/is valid yaml.